pom.xml

```diff
@@ -462,6 +462,7 @@
   </dependencyManagement>
 
   <build>
+    <defaultGoal>clean verify apache-rat:check javadoc:javadoc</defaultGoal>

```

Comment:
Hi,

I'd rather not generate the javadocs as part of the default build, doesn't make much sense to me. Also, on the build cheatsheet we're stating they're not generated, WDYT?

Reply:
changed.